index.html

```diff
@@ -15,14 +15,15 @@
        <li><a href="About.html">About</a></li>
        <li><a href="Gallery.html">Gallery</a></li>
        <li><a href="Services.html">Services Information</a></li>
-       <li><a href="Contact.html">Contacts</a></li>
-
-
-           <a href="" onclick="myFunction()"> <img id="popup" src="images/cart.jpeg">  </a>
-           
-        </div>
-
-        
+       <li><a href="Contact.html">Contacts</a></li> 
+       </ul>
+       </div>
+       
+        <div class="popup" onclick="myFunction()">
```

Comment:
Nice!  I would recommend you make the function name something more aligned to what the function does.  showCartPopup() perhaps?